tests/scripts/thread-cert/node.py

```diff
@@ -2724,6 +2724,10 @@ def ping(self, *args, **kwargs):
         else:
             return super().ping(*args, **kwargs)
 
+    def udp_send(self, ipaddr, port, data):
```

Comment:
This may hide `NodeImpl.udp_send` for `OtbrNode`:

```python
class OtbrNode(LinuxHost, NodeImpl, OtbrDocker):
```

Follow the current convention used by `LinuxHost.ping`, `LinuxHost.ping_ether` and `NodeImpl.ping` ?